tests/test_backend.py

```diff
@@ -1117,6 +1122,37 @@ def test_actual_expiry(self):
 
 
 class TestDefaultClient(unittest.TestCase):
+    def setUp(self):
+        self.client = caches[DEFAULT_CACHE_ALIAS].client
+
+    def tearDown(self):
+        try:
+            self.client.clear()
+        except exceptions.ConnectionInterrupted:
```

Comment:
No exception should pass silently, why is this here?

Reply:
hem...